test/plugins/retry5xx.js

```diff
@@ -664,5 +778,77 @@ describe('Retry5xx Plugin', function() {
           done();
         });
     });
+
+    it('successfully retries request with piped payload on 500 response and returns 200 response', function(done) {
```

Comment:
It'd be so much nicer to have this test in a function or something instead of pasted 3 times!
I'd still rather have it pasted 3 times than no test though :)